class/trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    trimmed = @nameable.correct_name[0, 10]
```

Comment:
- [ ] Remember string is index-based and it starts from 0, so to get 10, it would be from 0 to 9.